frontend-react/src/components/FileHandlers/FileHandlerMessaging.tsx

```diff
@@ -220,21 +217,115 @@ export const FileWarningBanner = ({ message }: FileWarningBannerProps) => {
     );
 };
 
+const HL7_PRODUCT_MATRIX_URL =
+    "https://www.hl7.org/implement/standards/product_brief.cfm";
+const CDC_LIVD_CODES_URL = "https://www.cdc.gov/csels/dls/livd-codes.html";
+
+export type ValidationErrorMessageProps = {
+    errorCode: ErrorCode;
+    field?: string;
+    message?: string;
+};
+
+export function ValidationErrorMessage({
+    errorCode,
+    field,
+    message,
+}: ValidationErrorMessageProps) {
+    let child: ReactNode;
+
+    switch (errorCode) {
```

Comment:
We could also use a string-to-JSX parser to replace variables/components so as to simplify this and preserve the `<error_code, description>` dictionary, but I'd rather go with this for now because it's more explicit, it allows for more flexibility with markup, and the errors are less likely to change going forward.